tests/scripts/ci_perf.py

```diff
@@ -0,0 +1,177 @@
+parser.add_argument('-uploadToBenchview', dest='uploadToBenchview', default=False, choices=[True, False])
+
+#ambient jenkins properties
+parser.add_argument('-workspace', dest='workspace', default=os.path.abspath(os.getcwd()))
+parser.add_argument('-gitBranchWithoutOrigin', dest='gitBranchWithoutOrigin', default='FAKE_BRANCHNAME')
+parser.add_argument('-gitBranch', dest='gitBranch', default='origin/FAKE_BRANCHNAME')
+parser.add_argument('-gitCommit', dest='gitCommit', default='FAKE_GIT_COMMIT')
+
+
+
+##########################################################################
+# Helper Functions
+##########################################################################
+
+def log(message):
+    """ Print logging information
+    Args:
+        message (str): message to be printed
+    """
+
+    print('[%s]: %s' % (sys.argv[0], message))
+
+def is_supported_version() -> bool:
+    """ Checkes that the version of python is at least 3.5
+    Returns:
+        bool : true if python version is 3.5 or higher
+    """
+    return version_info.major > 2 and version_info.minor > 4
+
+def run_command(runArgs, environment):
+    """ Run command specified by runArgs
+    Args:
+        runargs (str[]): list of arguments for subprocess
+        environment(str{}): dict of environment variable
+    """
+    log('RUNNING COMMAND: ' + runArgs)
+    subprocess.check_call(runArgs, env=environment)
+
+##########################################################################
+# CI jobs
+##########################################################################
+
+class jenkins_properties:
+    def __init__(self, workspace, git_branch_without_origin, git_branch, git_commit):
+        self.workspace = workspace
+        self.git_branch_without_origin = git_branch_without_origin
+        self.git_branch = git_branch
+        self.git_commit = git_commit
+
+
+class coreclr_scenarios_args:
+    def __init__(self, is_pr, arch, os, jit_name, opt_level, benchview_commit_name, bv_upload_sas_token, upload_to_benchview):
+        self.is_pr = is_pr
+        self.arch = arch
+        self.os = os
+        self.jit_name = jit_name
+        self.opt_level = opt_level
+        self.benchview_commit_name = benchview_commit_name
+        self.bv_upload_sas_token = bv_upload_sas_token
+        self.upload_to_benchview = upload_to_benchview
+
+def coreclr_scenarios(args, props):
+    architecture = args.arch
+    testEnv = ''
+    configuration = 'Release'
+    runType = 'private' if args.is_pr else 'rolling'
+    benchViewName = 'CoreCLR-Scenarios private ' + args.benchview_commit_name if args.is_pr else 'CoreCLR-Scenarios rolling ' + props.git_branch_without_origin + ' ' + props.git_commit
+    uploadString = '-uploadToBenchview' if args.upload_to_benchview else ''
+    
+    run_command('powershell -NoProfile wget https://dist.nuget.org/win-x86-commandline/latest/nuget.exe -OutFile \"' + props.workspace + '\\nuget.exe\"', os.environ)
```

Comment:
I'm not sure if you want to do this. Most developers have nuget on their machines. I would prefer an option to download it if needed, and place it under `%WORKSPACE%\bin\nuget`

Reply:
This is what perf.groovy does and the script replicates it:

def benchViewName = isPR ? 'CoreCLR-Scenarios private %BenchviewCommitName%' : 'CoreCLR-Scenarios rolling %GIT_BRANCH_WITHOUT_ORIGIN% %GIT_COMMIT%'
                        def uploadString = '-uploadToBenchview'

batchFile("powershell -NoProfile wget https://dist.nuget.org/win-x86-commandline/latest/nuget.exe -OutFile \"%WORKSPACE%\\nuget.exe\"")